bokun.yaml

```diff
@@ -447,6 +447,7 @@ paths:
         - activity
       operationId: getUpcomingAvailabilities
       summary: Get the next upcoming Activity availabilities
+      description: 'Get the list of all availabilities for the Activity id including sold-outs, if parameter "includeSoldOut"=true. Parameter "max" limits the number of results (still, results do not exceed 50).'
```

Comment:
This is not fully correct. `max` is only taken into consideration if `includeSoldOut=true`. Otherwise max is 50 by default.